### DISCUSSION TITLE
installer fail ? core tarballs arm64 and amd64

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4899] [Q&A]
https://github.com/TomTravis/crouton_Penguin/releases/tag/systemd_chromeOS
-u -r -t to update
I used test2
now PW needed to install
z account z pw on the core
the amd64 it is  ** sudo tar xzf ... install 
then sudo test2 -p -r -t -u
the arm64 uses sudo edit-chroot -r 
read the edit-chroot cmds

![Screenshot 2022-11-21 2 38 58 PM](https://user-images.githubusercontent.com/46386261/203172386-b229b4a9-7d4a-49ba-a5e1-418969f1edb2.png)

**Comment by Penguinbot4** [comment]
thank you
very nice of you mamba ?

**Reply by Penguinbot4** [reply]
Progress is being made  Crouton still does not install , debootstrap error not working.

usr/local ,exec,dev mount task namespace systemd via init enter-chroot in test
![Screenshot_2022-11-23_16-42-55](https://user-images.githubusercontent.com/118845597/203670063-7c03a85f-2bce-46ff-8816-07d554a9b7f6.png)
automated bot post

**Comment by Penguinbot4** [comment]
![Screenshot_2022-11-24_18-14-43](https://user-images.githubusercontent.com/118845597/203888520-abefa4b9-b0c9-4b1b-bf25-253eac50c718.png)
![Screenshot_2022-11-24_18-20-22](https://user-images.githubusercontent.com/118845597/203887536-1bb993df-3343-427c-b995-80af5280b6df.png)
yea gentoo and arch is installed
happy holiday 
penguinbot4 auto post

**Comment by Penguinbot4** [comment]
Arch core now working
name: ar2
encrypted: no
Entering /home/SD/4/chroots/ar2...
chromeOSLinux: version 1-20221123160824~master:f4fdd7e2 latest version 11-23-2022
works is on cras and xf86 install
still in development the shell scripts no longer work with crouton they have been heavily modified, @Drinkcat choargh does not work now, Just like a go install, tar and config, no debootstrap

[z@localhost ~]$ uname -a
Linux localhost 4.19.255-13653-ga5ef616743fe #1 SMP PREEMPT Thu Oct 13 19:09:54 PDT 2022 aarch64 GNU/Linux
[z@localhost ~]$

chronos@localhost /home/SD/4/chroots $ sudo enter-chroot -c /home/SD/4/chroots -n ar2 
Entering /home/SD/4/chroots/ar2...
[z@localhost ~]$ pacman -Q
acl 2.3.1-2
archlinux-keyring 20221110-1
archlinuxarm-keyring 20140119-2
argon2 20190702-4
attr 2.5.1-2
audit 3.0.8-1
base 3-1
bash 5.1.016-1
brotli 1.0.9-9
bzip2 1.0.8-5
ca-certificates 20220905-1
ca-certificates-mozilla 3.85-1
ca-certificates-utils 20220905-1
coreutils 9.1-3
cryptsetup 2.5.0-4
curl 7.86.0-4
db 5.3.28-5
dbus 1.14.4-1
device-mapper 2.03.17-1
dhcpcd 9.4.1-1
dialog 1:1.3_20220728-1
dnssec-anchors 20190629-3
e2fsprogs 1.46.5-4
expat 2.5.0-1
file 5.43-1
filesystem 2022.10.18-1
findutils 4.9.0-1
firmware-oak 1.0-1
freetype2 2.12.1-1
gawk 5.2.0-3
gcc-libs 12.1.0-2.1
gdbm 1.23-1
gettext 0.21.1-2
glib2 2.74.1-1
glibc 2.35-5.1
gmp 6.2.1-2
gnupg 2.2.40-1
gnutls 3.7.8-4
gpgme 1.18.0-1
graphite 1:1.3.14-2
grep 3.8-2
groff 1.22.4-7
gzip 1.12-1
harfbuzz 5.3.1-3
haveged 1.9.18-1
hwdata 0.364-1
iana-etc 20221107-1
icu 72.1-2
iproute2 6.0.0-1
iptables 1:1.8.8-2
iputils 20211215-1
iw 5.19-1
json-c 0.16-1
kbd 2.5.1-1
keyutils 1.6.3-1
kmod 30-3
krb5 1.20-3
ldns 1.8.3-2
less 1:608-1
libarchive 3.6.1-5
libassuan 2.5.5-1
libbpf 1.0.1-1
libcap 2.66-1
libcap-ng 0.8.3-1
libdrm 2.4.114-1
libedit 20210910_3.1-2
libelf 0.187-2
libepoxy 1.5.10-1
libevdev 1.13.0-1
libevent 2.1.12-4
libffi 3.4.4-1
libfontenc 1.1.6-1
libgcrypt 1.10.1-2
libglvnd 1.5.0-1
libgpg-error 1.46-2
libgudev 237-2
libidn2 2.3.4-3.1
libinput 1.22.0-1
libksba 1.6.2-1
libldap 2.6.3-2
libmnl 1.0.5-1
libnetfilter_conntrack 1.0.9-1
libnfnetlink 1.0.2-1
libnftnl 1.2.3-1
libnghttp2 1.51.0-1
libnl 3.7.0-1
libomxil-bellagio 0.9.3-3
libp11-kit 0.24.1-1
libpcap 1.10.1-2
libpciaccess 0.17-1
libpipeline 1.5.7-1
libpng 1.6.39-1
libpsl 0.21.1-3
libsasl 2.1.28-3
libseccomp 2.5.4-1
libsecret 0.20.5-2
libssh2 1.10.0-3
libsysprof-capture 3.46.0-1
libtasn1 4.19.0-1
libtirpc 1.3.3-2
libunistring 1.1-2
libunwind 1.6.2-2
libverto 0.3.2-4
libwacom 2.5.0-1
libx11 1.8.2-2
libxau 1.0.10-2
libxcb 1.15-2
libxcrypt 4.4.33-1
libxcvt 0.1.2-1
libxdamage 1.1.5-5
libxdmcp 1.1.4-1
libxext 1.3.5-1
libxfixes 6.0.0-2
libxfont2 2.0.6-1
libxkbfile 1.1.1-1
libxml2 2.10.3-2
libxrandr 1.5.3-1
libxrender 0.9.11-1
libxshmfence 1.3.1-1
libxxf86vm 1.1.5-1
licenses 20220125-1
linux-api-headers 5.18.15-1
linux-firmware 20221109.60310c2-2
linux-firmware-whence 20221109.60310c2-2
linux-oak 3.18.0-12
llvm-libs 14.0.6-4
lm_sensors 1:3.6.0.r41.g31d1f125-2
lz4 1:1.9.4-1
man-db 2.11.1-1
mesa 22.2.3-1
mpfr 4.1.1-1
mtdev 1.1.6-2
nano 7.0-1
ncurses 6.3-3
net-tools 2.10-1
netctl 1.28-1
nettle 3.8.1-1
npth 1.6-3
openresolv 3.12.0-1
openssh 9.1p1-3
openssl 3.0.7-2
p11-kit 0.24.1-1
pacman 6.0.2-5
pacman-mirrorlist 20220329-1
pam 1.5.2-1
pambase 20221020-1
pciutils 3.9.0-1
pcre 8.45-3
pcre2 10.40-3
perl 5.36.0-1
pinentry 1.2.1-1
pixman 0.42.2-1
popt 1.19-1
procps-ng 3.3.17-1
psmisc 23.5-1
readline 8.2.001-1
sed 4.8-1
shadow 4.12.3-2
sqlite 3.40.0-1
systemd 252.1-2
systemd-libs 252.1-2
systemd-sysvcompat 252.1-2
tar 1.34-1
tpm2-tss 3.2.0-3
tzdata 2022f-1
util-linux 2.38.1-1
util-linux-libs 2.38.1-1
vi 1:070224-6
vulkan-icd-loader 1.3.235-1
wayland 1.21.0-1
which 2.21-5
wireless-regdb 2022.08.12-1
wireless_tools 30.pre9-3
wpa_supplicant 2:2.10-7
xcb-proto 1.15.2-2
xf86-input-libinput 1.2.1-1
xkeyboard-config 2.37-1
xorg-fonts-encodings 1.0.6-1
xorg-server 21.1.4-1
xorg-server-common 21.1.4-1
xorg-setxkbmap 1.3.3-1
xorg-xkbcomp 1.4.5-1
xorg-xrandr 1.5.1-2
xorgproto 2022.2-1
xz 5.2.8-1
zlib 1:1.2.13-2
zstd 1.5.2-7

**Comment by Penguinbot4** [comment]
![Screenshot_2022-11-26_14-41-08](https://user-images.githubusercontent.com/118845597/204119163-0b9be0a7-556d-46fc-9b94-b79abfe12eba.png)

**Comment by Penguinbot4** [comment]
it is not broken
![Screenshot_2022-11-26_21-36-30](https://user-images.githubusercontent.com/118845597/204134008-e47bccfc-7155-4be7-a9ce-522e1cd110fa.png)

**Comment by Penguinbot4** [comment]
openSUSE 15.4 in progress
![Screenshot_2022-11-27_19-43-13](https://user-images.githubusercontent.com/118845597/204194296-b2f2125f-1619-40c9-a65f-cbd6f9ab0ef7.png)
